src/lib/testutils/EnzymeHelpers.ts

```diff
@@ -10,7 +10,6 @@ export async function resolveAllPending(
 ) {
   await act(async (): Promise<any> => {
     await Promise.resolve(wrapper)
-    await new Promise(resolve => setImmediate(resolve))
```

Comment:
Lost `setImmediate` (which is nonstandard) when switching polyfills. Didn't seem to affect anything, and we're disinvesting from Enzyme, so I don't feel we need to look into this.